src/components/PurchaseOrder/CloseOrder/CloseOrderModal.js

```diff
@@ -51,11 +63,42 @@ class CloseOrderModal extends Component {
         >
           <FormattedMessage id="ui-orders.paneBlock.closeBtn" />
         </Button>
-        <CloseOrderModalForm
-          close={this.closeModal}
-          orderId={orderId}
+        <Modal
+          label={<FormattedMessage id="ui-orders.closeOrderModal.title" values={{ orderId }} />}
           open={this.state.openModal}
-        />
+        >
+          <Row>
+            <Col xs={12}>
+              <Select
+                placeHolder="---Choose the reason---"
+                dataOptions={reasons}
+                required
+                label={<FormattedMessage id="ui-orders.closeOrderModal.reason" />}
+                onChange={(e => this.setState({ values: { reason: e.target.value } }))}
```

Comment:
please describe the state default values in constructor. And use specific attributes for reason and note, no need to use nesting and object like `values`.